schemas/process.yml

```
@@ -42,6 +42,31 @@
         Process id.
       example: 4242
 
+    - name: entity_id
+      level: extended
+      type: keyword
+      short: Unique identifier for the process.
+      description: >
+        Unique identifier for the process.
+
+        The implementation of this is specified by the data source, but some
+        examples of what could be used here are a process-generated UUID,
+        Sysmon Process GUIDs, or a hash of some uniquely identifying components
+        of a process.
```

Love this. Should we add another sentence about the "why" as well? Example:

```suggestion
        of a process.

        This is a common practice, to mitigate PID reuse, and get a more unique identifier for processes over time, and across monitored hosts.
```

Not absolutely necessary to add: people who know will understand what this is. But it's a good opportunity to give good ideas to people who hadn't thought of that :-)

---

Sounds good, added a variation of the suggested text.